io.github.zzzgydi.clash_verge.desktop

```diff
@@ -0,0 +1,8 @@
+[Desktop Entry]
+Categories=Development;
```

Comment:
this is development tool?

Reply:
Yes, the upstream .desktop file is like this, this software is very special, most of the people who use this software use the Chinese to bypass the GFW, so this software cannot be explicitly said to be proxy software (I guess)

Reply:
that doesn't explain why the file is here.

Reply:
This .desktop file should be tauri packaged so that it is automatically generated

Reply:
I have changed category to Network now